src/containers/withUser.js

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'redux'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->